packages/app/src/request.ts

```diff
@@ -16,16 +16,16 @@ export { getURLParams } from '@tinyhttp/req'
 
 const trustRemoteAddress = ({ socket }: Pick<Request, 'headers' | 'socket'>) => {
   const val = socket.remoteAddress
-  if (typeof val === 'string') return compile(val.split(',').map((x) => x.trim()))
-  return compile(val || [])
+  if (typeof val !== 'string') return compile(val || [])
```

Comment:
why swap it? it's the same logic

Reply:
`socket.remoteAddress` is always a string so it would never reach line 20

Reply:
then return it directly

Reply:
looks like it's a string https://nodejs.org/docs/latest/api/net.html#socketremoteaddress